app/views/layouts/application.html.erb

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en" class="govuk-template">
+<html lang="en">
   <head>
     <meta charset="utf-8" />
     <title><%= t('coronavirus_form.errors.page_title_prefix') if flash[:validation]&.any? %><%= yield :title %> - GOV.UK</title>
```

Comment:
We need a task to adjust the header meta asap.

Reply:
I've created subtask 4050 to do this @markheathy

Reply:
👍